docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -3,19 +3,36 @@
 include::_attributes.adoc[]
 :categories: security,web
 
-The Quarkus Security framework supports multiple authentication mechanisms, which you can use to secure your applications. 
-Some of the supported authentication mechanisms are built into Quarkus and some require you to add an extension.
-You can also combine authentication mechanisms. 
+The Quarkus Security framework supports multiple authentication mechanisms, which you can use to secure your applications.
+Some supported authentication mechanisms are built into Quarkus, and some require you to add an extension.
+You can also combine authentication mechanisms.
 
 Before you choose an authentication mechanism for securing your Quarkus applications, review the information provided.
 
+.Authentication requirements and mechanisms
+|===
+^|Authentication Requirement ^| Authentication Mechanism ^| Notes
+
+^|Username and password ^|xref:security-basic-authentication-concept.adoc[Basic], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form] ^|
+
+^|Bearer Access Token  ^| xref:xref:security-openid-connect-web-authentication.adoc[OIDC], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]  ^|See the <<table>> table below.
```

Comment:
This link should point to the bearer access token doc, currently named `security-openid-connect.adoc`

Reply:
I see, double `xref` here

Reply:
Fixing

Reply:
@MichalMaler The other thing is that `Notes` column seems redundant right now, it was my idea I know, but it does not look good yet while we only have a single note, as it creates a widest column, with all but only a single value set, my proposal is to collapse that note into the 2nd column `OIDC, JWT, OAuth2 (see <<table>> table below)`

Reply:
Yeah, I already tried that. The problem is that it doenst look well and asciidoc doesnt allowed me to use bullet point for that Table-note. Need to find a way how to do it. Working on it

Reply:
Nice, found a solution. Adding.

Reply:
@sberyozkin by that file `security-openid-connect.adoc`, did you meant `security-openid-connect-client.adoc` ?

Reply:
it was this one I guess `security-oidc-bearer-authentication-concept.adoc` Renamed and recomposed by Rolfe.